Formula/socket_vmnet.rb

```diff
@@ -25,6 +25,13 @@ def install
     system "make", "install.bin", "install.doc", "VERSION=#{version}", "PREFIX=#{prefix}"
   end
 
+  def post_install
+    # We want root to take ownership of `var/"run/socket_vmnet"`,
+    # so we don't create it ourselves.
+    (var/"run").mkpath
+    (var/"log/socket_vmnet").mkpath
+  end
+
   def caveats
```

Comment:
This will also be printed by brew so can be removed

Reply:
brew doesn't print recommendation for `sudo` AFAICS

Reply:
Yes it does, because of `require_root` in the service block

Reply:
It doesn't make clear that `socket_vmnet` _must_ be run with `sudo`, though:
```
To restart socket_vmnet after an upgrade:
  sudo brew services restart socket_vmnet
```
I just added the usual `caveats` from other formulae that require root (e.g. `bettercap`, `boringtun`, `htop`, `sntop`, etc...).

Reply:
That's if it detects as running already.

Reply:
That's interesting, since I hadn't even installed it yet

Reply:
Here's the complete output:
```
==> socket_vmnet: stable 1.1.1 (bottled), HEAD [keg-only]
Daemon to provide vmnet.framework support for rootless QEMU
https://github.com/lima-vm/socket_vmnet
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/socket_vmnet.rb
License: Apache-2.0
==> Requirements
Required: macOS ✔, macOS >= 10.15 (or Linux) ✔
==> Options
--HEAD
        Install HEAD version
==> Caveats
To install an optional launchd service, run the following command (sudo is necessary):
sudo brew services start socket_vmnet

socket_vmnet is keg-only, which means it was not symlinked into /usr/local,
because /usr/local/bin is often writable by a non-admin user.


To restart socket_vmnet after an upgrade:
  sudo brew services restart socket_vmnet
Or, if you don't want/need a background service you can just run:
  /usr/local/opt/socket_vmnet/bin/socket_vmnet --vmnet-gateway=192.168.105.1 /usr/local/var/run/socket_vmnet
==> Analytics
install: 116 (30 days), 664 (90 days), 773 (365 days)
install-on-request: 116 (30 days), 662 (90 days), 771 (365 days)
build-error: 0 (30 days)
```
Notice: `Not installed`

Reply:
Incidentally, the `brew services` message is shown only if you're not installing from the API.
```
❯ HOMEBREW_NO_INSTALL_FROM_API= brew info socket_vmnet
==> socket_vmnet: stable 1.1.1 (bottled), HEAD [keg-only]
Daemon to provide vmnet.framework support for rootless QEMU
https://github.com/lima-vm/socket_vmnet
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/socket_vmnet.rb
License: Apache-2.0
==> Requirements
Required: macOS ✔, macOS >= 10.15 (or Linux) ✔
==> Options
--HEAD
        Install HEAD version
==> Caveats
To install an optional launchd service, run the following command (sudo is necessary):
sudo brew services start socket_vmnet

socket_vmnet is keg-only, which means it was not symlinked into /usr/local,
because /usr/local/bin is often writable by a non-admin user.

==> Analytics
install: 116 (30 days), 664 (90 days), 773 (365 days)
install-on-request: 116 (30 days), 662 (90 days), 771 (365 days)
build-error: 0 (30 days)
```
That should explain why @AkihiroSuda doesn't see the message. But I think that's Homebrew/brew#14925.

Reply:
Let's keep the caveat for now then